src/pycountry/db.py

```diff
@@ -53,18 +57,21 @@ def __init__(self, filename):
         self._is_loaded = False
         self._load_lock = threading.Lock()
 
+        # create data class
```

Comment:
I moved the definition of the data class to the constructor. This makes sure the data type is defined even before the database is loaded. That makes it easier to reason about object types at runtime, and could potentially enable type hints like the below (not tested):

```python
def get_last_visited_country() -> pycountry.countries.data_class:
    ...
```

Reply:
On that note, I think it could actually be beneficial to define the data classes outside of the Database, so this can be referenced even more directly:

```python
def get_last_visited_country() -> pycountry.Country:
    ...
```

However, I didn't want to introduce too much change in my PR, and type hinting was not my focus.